RadixWallet/Core/DesignSystem/Components/PoolUnitView.swift

```diff
@@ -46,51 +46,49 @@ public struct PoolUnitView: View {
 
 					Spacer(minLength: 0)
 
+					if let isSelected = viewState.isSelected {
+						CheckmarkView(appearance: .dark, isChecked: isSelected)
+					}
+
 					//					AssetIcon(.asset(AssetResource.info), size: .smallest)
```

Comment:
time to remove dead code below?

Reply:
This is actually upcoming code, it will be activated once we have the copy for those info views